exporter/coralogixexporter/config.go

```diff
@@ -44,8 +47,34 @@ type Config struct {
 }
 
 func (c *Config) Validate() error {
-	if c.Endpoint == "" || c.PrivateKey == "" || c.AppName == "" || c.SubSystem == "" {
+	// check env variables
+	endpoint := c.Endpoint
+	privateKey := c.PrivateKey
+	appName := c.AppName
+	subSystem := c.SubSystem
+
+	if os.Getenv("CORALOGIX_ENDPOINT") != "" {
+		endpoint = os.Getenv("CORALOGIX_ENDPOINT")
+	}
+	if os.Getenv("CORALOGIX_PRIVATE_KEY") != "" {
+		privateKey = os.Getenv("CORALOGIX_PRIVATE_KEY")
+	}
+	if os.Getenv("CORALOGIX_APPLICATION_NAME") != "" {
+		appName = os.Getenv("CORALOGIX_APPLICATION_NAME")
+	}
+	if os.Getenv("CORALOGIX_SUBSYSTEM_NAME") != "" {
+		subSystem = os.Getenv("CORALOGIX_SUBSYSTEM_NAME")
+	}
+	if endpoint == "" || privateKey == "" || appName == "" || subSystem == "" {
 		return fmt.Errorf("One of your configureation field (endpoint, private_key, application_name, subsystem_name) is empty, please fix the configuration file")
```

Comment:
You might want to take the opportunity and fix the error message as well.

Reply:
Thanks, I will fix the error message